galata/src/galata.ts

```diff
@@ -29,9 +29,14 @@ export namespace galata {
       editorConfig: { cursorBlinkRate: 0 }
     },
     '@jupyterlab/notebook-extension:tracker': {
-      codeCellConfig: { cursorBlinkRate: 0 },
-      markdownCellConfig: { cursorBlinkRate: 0 },
-      rawCellConfig: { cursorBlinkRate: 0 }
+      //codeCellConfig: { cursorBlinkRate: 0 },
+      //markdownCellConfig: { cursorBlinkRate: 0 },
+      //rawCellConfig: { cursorBlinkRate: 0 },  // this should make some tests fail (checking if this makes a difference)
```

Comment:
FYI The `cursorBlinkRate` impact is hard to catch as this makes the tests flaky. Every tests could still be passing if you are lucky enough that all snapshots got generated when the cursor was visible (we had that before).